gcloud-java-examples/pom.xml

```diff
@@ -37,6 +42,38 @@
           <skip>false</skip>
         </configuration>
       </plugin>
+        <!-- make a script so we can run stuff directly by calling
+             mvn package appassembler:assemble -DskipTests -Dmaven.javadoc.skip=true  -Dmaven.source.skip=true
+             from the gcloud-java-examples folder. The script is written to target/appassembler/bin/. -->
```

Comment:
I added this as a list item on the "writing a new service" doc issue; will include when I write the doc.